### ISSUE TITLE
Kul Tiras Marine dont have CD on Shield Bash

### ISSUE THREAD

**Issue by laytya** [Wall-core/Everlook-Bugtracker#189]
![nocd](https://user-images.githubusercontent.com/20452393/194740192-56a1b593-8843-4418-be70-cdfa1ef1c63e.PNG)

2 sec its not CD

**Comment by Alakoris** [comment]
To be more accurate, they have 2 versions of that ability, which don't share same cooldown. After your skill refreshed, they can't use it anymore, which means it is still on CD. Please, remover one of the skills ^^

**Comment by Tessablanc** [comment]
No, they don't. They have one Shield Bash ability and that's all: https://vanilla-twinhead.twinstar.cz/?npc=3129#abilities

Shield Bash currently has no cooldown on any mob, apparently.

**Comment by laytya** [comment]
it seems Shield Bash have CD - i saw it on other mobs. But Kul Tiras Marine have 2 of them w/ different CD... so they do 2 SB and after it on CD both.
